src/Core/src/Handlers/Editor/EditorHandler.iOS.cs

```diff
@@ -77,6 +77,20 @@ public override Size GetDesiredSize(double widthConstraint, double heightConstra
 			return base.GetDesiredSize(widthConstraint, heightConstraint);
 		}
 
+		public override bool NeedsContainer
+		{
+			get
+			{
+				// If we are wrapped by a BorderView, we need a container
+				// to handle masks and clip shapes effectively.
+				if (VirtualView?.Parent is IBorderView)
```

Comment:
I don't feel like this is the approach we'll want to take here. 

1) Users report that this all works fine with the Frame control
2) it doesn't feel like Editor should be reaching up to the parent here to figure out if it needs a container.  This feels like it's not fixing the root of the problem. 

I'm still poking a bit to see if I can figure out a better approach here. 

I'm curious if there's a way to fix this inside the BorderView control vs fixing here